### DISCUSSION TITLE
so uh.. protondb plugin github is a 404...?

### DISCUSSION THREAD

**Discussion by GodsBadAssBlade** [CrankshaftForSteam/crankshaft#34] [General]
the plugin i use at a glance and for quick access to the specific games db no longer functions and when clicking to see the source code the github page responds with a 404.. what now?

**Comment by DrakiaXYZ** [comment]
I'm not seeing a 404 on the ProtonDB indicator repo:
https://git.sr.ht/~avery/protondb-indicator

However currently the latest Stable release of the SteamOS software has broken CrankShaft. Avery is working on a fix, but we don't have an ETA.

**Reply by GodsBadAssBlade** [reply]
> I'm not seeing a 404 on the ProtonDB indicator repo: https://git.sr.ht/~avery/protondb-indicator
> 
> However currently the latest Stable release of the SteamOS software has broken CrankShaft. Avery is working on a fix, but we don't have an ETA.

oh really? must be the link to the source on crankshafts end then, glad to hear it'll be back eventually though!
